commons/commands.py

```diff
@@ -573,6 +573,8 @@
 KUBECTL_DESCRIBE_POD_CMD = "kubectl describe pod {}"
 KUBECTL_GET_STATEFULSET = "kubectl get sts | grep '{}'"
 KUBECTL_CREATE_STATEFULSET_REPLICA = "kubectl scale statefulset {} --replicas {}"
+KUBECTL_GET_POD_PORTS = "kubectl get pods cortx-server-0 -o jsonpath='{{.spec.containers[" \
```

Comment:
why not make command generic without giving cortx-server-0

Reply:
That was a miss, thank for noticing, will update it.